src/messages/TextWithButtons/TextWithButtons.tsx

```diff
@@ -55,13 +55,20 @@ const TextWithButtons: FC = (props: ITextWithButtonsProps) => {
 	);
 
 	useEffect(() => {
+		const messageConfig = message.data?._cognigy?._webchat as IWebchatMessage;
+		const textInQuickReplies = messageConfig?.message?.text;
+		const textInButtons = (messageConfig?.message?.attachment as IWebchatTemplateAttachment)
+			?.payload?.text;
```

Comment:
This logic currently covers only the Webchat channel. 
See the lines 50-55 in this file for the broader approach which includes Facebook and Default channels.

Reply:
Ya, right. I hope I can they reuse the text const that we already define